File: LV1/1.4/babybites/babybites.py
_ = input()
l = input().split()
cnt = 1

is_valid = True
for i in l:
    if i != "mumble" and i != str(cnt):
        is_valid = False
        break
    cnt += 1

if is_valid:
    print("makes sense")
else:
    print("something is fishy")

File: LV1/1.4/babybites/babybites.cpp
#include <cstdio>
#include <cstdlib>

int main() {
    int n;
    scanf("%d", &n);

    char s[10];
    int cnt = 1;
    bool is_make_sense = true;

    for (int i = 0; i < n; i++) {
        scanf("%s", s);

        if (s[0] != 'm') {
            int num = atoi(s);
            if (num != cnt) {
                is_make_sense = false;
                break;
            }
        }
        cnt++;
    }

    if (is_make_sense) {
        puts("makes sense");
    } else {
        puts("something is fishy");
    }
}
